packages/web3torrent/src/clients/payment-channel-client.ts

```diff
@@ -209,6 +209,8 @@ export class PaymentChannelClient {
       channelState.channelId === channelId && channelState.status === 'closed';
 
     if (!['closing', 'closed'].includes(this.channelCache[channelId].status)) {
+      await this.getLatestPaymentReceipt(channelId);
+      log.info('My turn, about to close channel');
```

Comment:
This doesn't look like it's intended to get into master.

Reply:
https://github.com/statechannels/monorepo/pull/1969/commits/034e40f790457b167989f3792ebeda37eaecdbb4